### PR TITLE
NEXT-32862 - Add docs for default FS adapter config fallback

### PR DESCRIPTION
**Pull request by keulinho**

### CODE REVIEW

File: guides/hosting/infrastructure/filesystem.md [hunk 1]
```diff
@@ -53,6 +53,13 @@ shopware:
       # The Adapter Configuration
 ```
 
+### Fallback adapter configuration
+
+By default, the configuration for the theme, asset and sitemap filesystem will use the configuration from the `public` filesystem, if they are not specifically configured.
+This means when you want to change the configuration used for the public filesystem, but the others should use the old configuration you have to set them explicitly.
```
Comment (shyim) [inline guides/hosting/infrastructure/filesystem.md:59]: maybe add an example with yaml anchor to copy the config 🤔

File: guides/hosting/infrastructure/filesystem.md [hunk 2]
```diff
@@ -53,6 +53,13 @@ shopware:
       # The Adapter Configuration
 ```
 
+### Fallback adapter configuration
+
+By default, the configuration for the theme, asset and sitemap filesystem will use the configuration from the `public` filesystem, if they are not specifically configured.
```
Comment (sushmangupta) [inline guides/hosting/infrastructure/filesystem.md:58]:
```suggestion
By default, the configuration for the theme, asset and sitemap filesystem will use the configuration from the `public` filesystem if they are not specifically configured.
```

File: guides/hosting/infrastructure/filesystem.md [hunk 3]
```diff
@@ -53,6 +53,13 @@ shopware:
...
+This means when you want to change the configuration used for the public filesystem, but the others should use the old configuration you have to set them explicitly.
+
+### Additional configuration
+
 If you want to regulate the uploaded file types, then you could add the keys `allowed_extensions`for the public filesystem or `private_local_download_strategy` for the private filesystem.
```
Comment (sushmangupta) [inline guides/hosting/infrastructure/filesystem.md:63]:
```suggestion
If you want to regulate the uploaded file types, then you could add the keys `allowed_extensions` for the public filesystem or `private_local_download_strategy` for the private filesystem.
```

**Review by sushmangupta — CHANGES_REQUESTED** [review]

**Review by sushmangupta — APPROVED** [review]
Thank you for the update! :)

**Pull request merged**